### ISSUE TITLE
Staging Review finding: Focus behavior on page transition doesn't match other pages

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91816]
## Value Proposition

As a veteran using a screen reader to navigate through the CST, I want the screen reader to read the most important heading on the page first upon page load, so that I can effectively orient myself to the content on the page. 

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tools
**Product:** Claim Status Tool
**Feature:** Improved 5103 requests and waiver experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a user navigates through the pages related to a given claim (status, files, overview), keyboard focus moves to the H2 for each page. This puts keyboard-only users in the right place for each page without having to excessively tab, and gives screen reader users immediate information about where they are.

But when a user navigates to the 5103 Evidence Notice page, focus goes to the breadcrumbs. Since the page is structured differently than status/files/overview are, there's no corresponding H2 -- but there is a unique H1 that plays a similar role describing the page.

The impact on keyboard-only users is manageable --- extra keystrokes to tab through the breadcrumbs, but it's not too bad. The impact to screen reader users is a little bigger, since it interrupts the mental model you build when moving through the product. It doesn't take too much effort to figure out where you are, but it does require you to put in the effort to figure out where you are.

<details><summary>Link, screenshot or steps to recreate</summary>

1. Log in with test user Kyle Cole and navigate to an appropriate claim.
2. Tab to "Files" and press [enter].
3. Observe that focus moves to the "Claim files" H2.
4. Tab to "Details" in the 5103 alert and press [enter].
5. Observe that focus moves to the breadcrumbs.

</details>

## Recommended action

Ideally, I'd like to see focus move to the H1 when navigating to the 5103 Evidence Notice page.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

## Acceptance Criteria

- [ ] Upon page load, focus is set to the first H1 on the document request page
- [ ] Upon page load, focus is set to the first H1 on the claim letter page

**Comment by davidakennedy** [comment]
This is a good callout. @briandeconinck You and I talked about this a bit yesterday, one thought I had as I looked at this again today: When you load the initial Claim Status Tool at `https://staging.va.gov/track-claims/your-claims`, focus goes to the breadcrumbs. I think that's why I was okay with the breadcrumbs here. If we move focus to the heading here, does it make sense to do that on the `your-claims` page. Makes sense to me, but wanted to get your thoughts.

For our engineers, we want to move the focus to the h1 in the screenshot when this page loads.

![Evidence review page with red arrow pointing to h1 with text that says move focus here.](https://github.com/user-attachments/assets/e898a397-be9e-48b5-ad20-de189fe21385)

**Comment by jacobworrell** [comment]
@davidakennedy is the recommendation to do that just for the 5103 Evidence pages (and other evidence request pages) or should this be standard for all CST pages?

**Comment by davidakennedy** [comment]
@jacobworrell For the purposes of this ticket, and getting launched, the most important part is moving focus to the h1 of the evidence notice page.

If we want to make things more consistent, I think we could also move focus to the h1 of the `/track-claims/your-claim-letters` (Letters) page. Right now, focus lands on the breadcrumbs. I would leave focus of of the main claims status page on the breadcrumbs because we often don't know where folks are coming from.

**Comment by bacitracin** [comment]
Evidence request pages have been revamped recently and focus goes to h1 now
